src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/database/daos/ResourceMetadataDao.kt

```diff
@@ -115,6 +115,29 @@ class ResourceMetadataDao(
                 }
     }
 
+    fun fetchByLanguageAndIdentifier(languageSlug: String, identifier: String, dsl: DSLContext = instanceDsl): ResourceMetadataEntity? {
```

Comment:
This.... also might not be enough to get a unique metadata entry. I don't remember exactly what we decided the unique constraint to be, but version may come into play as well.

Reply:
[UNIQUE (language_fk, identifier, version, creator, derivedFrom_fk)](https://github.com/WycliffeAssociates/otter-jvm/blob/84520bd367b98259643e2b0932a9bb2c019c6dcb/src/main/resources/sql/CreateAppDb.sql#L27)

Reply:
Hm. I know we had talked about version previously and the fact that the version numbers might not match up between related resource containers? Really not sure how to handle this for now though.